cartridges/int_algolia/cartridge/scripts/algolia/helper/jobHelper.js

```diff
@@ -43,6 +43,96 @@ function xmlToArray(xmlArray) {
     return result;
 }
 
+/**
+ * Check object for empty
+ * @param {Object} obj - any Object
+ * @returns {boolean} - is empty Object
+ */
+function _isEmptyObject(obj) {
+    if (obj instanceof Object) {
+        var keys = Object.keys(obj);
+        for (var i = 0; i < keys.length; i += 1) {
+            if (Object.hasOwnProperty.call(obj, keys[i])) return false;
+        }
+        return true;
+    }
+    return empty(obj);
+}
+
+/**
+ * Function to create a clone object
+ * @param {Object} obj - object to clone
+ * @returns {Object} - clobe of Obj
+ */
+function _cloneObject(obj) {
```

Comment:
Not really, as these private methods are used in some of the methods used in the new code. If I put these in another PR, someone applying the changes from this PR would have to include that PR too.
Only completely unrelated changes could be moved to another PR.

Reply:
Where? I don't see it (And the renaming + move wasn't mandatory)

Reply:
The move was mandatory because some of these methods were used by another script internally and since I'm reusing them, I needed to move them out of that script into `jobHelper.js`.